.changeset/rich-rice-laugh.md

```diff
@@ -0,0 +1,8 @@
+---
+"near-api-js": minor
+---
+
+Add builder pattern transaction creation feature as an alternative way to create and send transactions:
+```typescript
+bob.createTransaction("alice.near").transfer("1000000000000000000").signAndSend();
```

Comment:
Could we make it more clear that `bob` is an instance of `Account` here?

Maybe something like
```ts
new Account(connection, 'bob.near').createTransaction("alice.near").transfer("1000000000000000000").signAndSend();
```

Reply:
Updated